activity-stream-impl/src/main/kubernetes-resources/platform.conf

```diff
@@ -0,0 +1,27 @@
+akka {
```

Comment:
A general theme - if the config was relatively static I moved it a target `platform.conf` - e.g. `kubernetes-resources/platform.conf` or `marathon/platform.conf`